platform-controller/src/test/java/org/hobbit/controller/docker/ContainerManagerImplTest.java

```diff
@@ -29,6 +29,7 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import org.hobbit.controller.containers.docker.ContainerManagerImpl;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.